spec/plaindate.html

```diff
@@ -730,6 +730,7 @@ <h1>
           1. If _sign_ × (_d1_ - _isoDate2_.[[Day]]) > 0, return *true*.
         1. Return *false*.
       </emu-alg>
+      <emu-note>This operation intentionally uses overflow ~constrain~ when regulating the year-month.</emu-note>
```

Comment:
Is it worth explaining why we use `~constrain~` here?

Reply:
I'm not sure I can clearly explain why. It's just a choice we made for how the algorithm works. There are other choices that could have been made.

Do you want to try?

Reply:
Well it was actually added by @sffc here: https://github.com/tc39/proposal-temporal/pull/3138

It's been too long since I read Shane's proof in the comments of that PR, so I don't have context on it anymore.

Reply:
Ok, I think the answer is that `.until` does not know anything about overflow options; only `.add` accepts such a parameter. As a result, `.until` returns a duration that subsequently works with `.add` in overflow constrain mode, but the duration might not work in overflow reject mode.